mautic/mautic
# [Feature] Contact details URL hyperlink

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #5784
| BC breaks? | No
| Deprecations? | 

[//]: # ( Required: )
#### Description:
Just wraps URL fields with link tags in details view.

#### Steps to test this PR:
1. Create a contact filling in a URL field.
2. Open the contact and click Details.
3. You should be able to click on the URL, opening in a new tab.
